lib/msf/ui/console/command_dispatcher/core.rb

```diff
@@ -1541,6 +1541,14 @@ def cmd_set(*args)
       @cache_payloads = nil
     end
 
+    # Raise an alert if LHOST is set to 127.0.0.1
+    name = args[0]
+    ip = args[1]
+
+    if (name.upcase == "LHOST" && ip == "127.0.0.1")
+      print_error("You may experience errors with this choice of address for LHOST.")
```

Comment:
Wherever we decide to put this check, if `print_warning` is available, please use that instead. Thanks.